src/index.js

```diff
@@ -822,20 +825,36 @@ class Offline {
               else if (integration === 'lambda-proxy') {
                 response.statusCode = statusCode = result.statusCode || 200;
 
```

Comment:
multiValueHeaders are also supported in the lambda-proxy response.  Unfortunately, I couldn't get Hapi to write multiple headers with the same name, so I'm using the 'append' option they provide to concatenate the values.